conda/recipes/librmm/recipe.yaml

```diff
@@ -0,0 +1,105 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+# Copyright (c) 2018-2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
```

Comment:
One more non-blocking question: Can we use `env[KEY]` since there are no defaults, or do we need `env.get(KEY)`?

Reply:
it seems like it works -- I think I'm inclined to keep the `get` so we better match `rattler` docs and also don't lead people into thinking that it supports arbitrary jinja

Reply:
Typically in Python, `dict.get` implies a default of `None`. Is that true here, too?

Reply:
No, if the variable isn't found it will error.

Reply:
Great. I'll resolve this.